### ISSUE TITLE
train relative not converge

### ISSUE THREAD

**Issue by puyiwen** [LiheYoung/Depth-Anything#156]
Hi, thank you for your work! I want to use Depth-Anything to train my own rgb datasets. I get the raw network depth output, like 0-697 float values. And then I normalized the raw depth output to [0-1] float values. I use depth estimation model to train the normalized values, but the model does not converge. The Loss I only use L1 loss, is there something wrong with this loss function? I dont know why, can you help me? Thank you very much!!

**Comment by puyiwen** [comment]
I am so sorry, I found that it was a problem with my original data processing. The issue will be closed
